src/phoenix/core/traces.py

```diff
@@ -36,17 +36,17 @@ def __init__(self) -> None:
 
     def get_project(self, project_name: str) -> Optional["Project"]:
         with self._lock:
-            return self._projects.get(project_name)
+            return self._projects.get(project_name, Project())
 
     def get_projects(self) -> Iterator[Tuple[int, str, "Project"]]:
         with self._lock:
             for project_id, (project_name, project) in enumerate(self._projects.items()):
                 if project.is_archived:
                     continue
-                yield project_id, project_name, project
+                yield project_id + 1, project_name, project
 
     def archive_project(self, id: int) -> Optional["Project"]:
-        if id == 0:
+        if id == 1:
             raise ValueError("Cannot archive the default project")
```

Comment:
this might be a slightly scary check. I guess we can assume this but I would at least put it in a onstant

Reply:
I guess we could lift this restriction overall if needed

Reply:
This is legacy code that will be removed, but for the time being the indexing is updated to match those from the database so things don't break (as we incrementally migrate things).